src/Microsoft.AspNet.Mvc.ViewFeatures/Controller.cs

```diff
@@ -424,6 +424,33 @@ public virtual JsonResult Json(object data, [NotNull] JsonSerializerSettings ser
         }
 
         /// <summary>
+        /// Creates a <see cref="HttpOkResult"/> object that produces an empty OK (200) response.
+        /// </summary>
+        /// <returns>The created <see cref="HttpOkResult"/> for the response.</returns>
+        [NonAction]
+        public virtual HttpOkResult Ok()
+        {
+            return new HttpOkResult();
+        }
+
+        /// <summary>
+        /// Creates an <see cref="HttpOkObjectResult"/> object that produces an OK (200) response.
+        /// </summary>
+        /// <param name="value">The content value to format in the entity body.</param>
+        /// <returns>The created <see cref="HttpOkObjectResult"/> for the response.</returns>
+        [NonAction]
+        public virtual HttpOkObjectResult Ok(object value)
+        {
+            var disposableValue = value as IDisposable;
+            if (disposableValue != null)
+            {
+                Response.RegisterForDispose(disposableValue);
```

Comment:
All of our `ObjectResult`-returning methods do this. There's no real other good place to put it because a filter could present your result from running.